src/cli/utils.js

```diff
@@ -49,3 +49,15 @@ exports.getIPFS = (callback) => {
 exports.getRepoPath = () => {
   return process.env.IPFS_PATH || os.homedir() + '/.ipfs'
 }
+
+exports.createLogger = (visible) => {
+  return (msg, newline = true) => {
```

Comment:
Oh, I was sure that js-ipfs was compiled, even when used from node. Is that not correct?